templates/static/js/example.js

```diff
@@ -270,6 +271,14 @@
     return scripts;
   }
 
+  function fixScroll() {
+    const titleId = window.location.hash;
+    if (titleId) {
+      const title = document.querySelector(`${titleId}`);
```

Comment:
`titleId` seems to be a string, so do we need the backticks?

Reply:
Ahh - good spot! I was trying a few diff solutions and forgot to change this back. Thanks!